### PR TITLE
New endpoint for getting all released datasets.

### PR DESCRIPTION
**Pull request by vaskalan**
This branch (and PR) is only for getting released datasets - please excuse misleading branch name.

### CODE REVIEW

File: tests/resources/SSB_FDB/datastore/datastore_versions.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         },
         {
           "description": "Deleted",
-          "name": "DELETED_DATASET",
+          "name": "DATASET_TO_DELETE",
```
Comment (DanielElisenberg) [inline tests/resources/SSB_FDB/datastore/datastore_versions.json:26]: This dataset is already status "DELETED" so this namechange seems confusing

File: datastore_version_manager/domain/datastore_versions.py [hunk 2]
```diff
@@ -37,5 +37,31 @@ def bump_datastore_versions(description: str, pending_ops: dict, release_time: i
...
+    released_datasets = []
+    deleted_datasets = []
+    for version_instance in version_instances:
+        for data_structure_update in version_instance["dataStructureUpdates"]:
+            if data_structure_update["releaseStatus"] == "DELETED":
+                deleted_datasets.append(data_structure_update["name"])
+                continue
+            if data_structure_update["name"] in deleted_datasets:
+                continue
+            if any(d['datasetName'] == data_structure_update["name"] for d in released_datasets):
```
Comment (DanielElisenberg) [inline datastore_version_manager/domain/datastore_versions.py:54]: d => released_dataset

File: datastore_version_manager/domain/datastore_versions.py [hunk 3]
```diff
@@ -37,5 +37,31 @@ def bump_datastore_versions(description: str, pending_ops: dict, release_time: i
     return data_structures_to_bump, new_version
 
 
+def get_released_datasets():
```
Comment (DanielElisenberg) [inline datastore_version_manager/domain/datastore_versions.py:40]: :100:

**Review by DanielElisenberg — APPROVED** [review]
Looks good! :100: Minor readability comments

File: tests/resources/SSB_FDB/datastore/datastore_versions.json (hunk 1 above)
Comment (vaskalan) [reply tests/resources/SSB_FDB/datastore/datastore_versions.json:26]: The idea is that the dataset is released in version 1.0.0.0 with the intension to delete it in next version, for testing purposes.

File: datastore_version_manager/domain/datastore_versions.py [hunk 4]
```diff
@@ -51,12 +51,13 @@ def get_released_datasets():
...
                 continue
-            if any(d['datasetName'] == data_structure_update["name"] for d in released_datasets):
+            if any(released_dataset['datasetName'] == data_structure_update["name"]
+                   for released_dataset in released_datasets):
                 continue
             released_datasets.append(
                 {
                     "datasetName": data_structure_update["name"],
-                    "version": version_instance["version"],
+                    "version": version_instance["version"][:-2],
```
Comment (DanielElisenberg) [inline datastore_version_manager/domain/datastore_versions.py:60]: :100:

**Review by DanielElisenberg — APPROVED** [review]

**Pull request merged**